docs/api/microcosm.md

```diff
@@ -5,13 +5,58 @@
 
 ## Overview
 
-Microcosm is [Flux](https://facebook.github.io/flux/) with first-class
-actions and state sandboxing.
+The Microcosm class provides a centralized place to store application
+state, dispatch actions, and track changes.
```

Comment:
I am wondering if we should keep the reference to Flux for the "This is like X" aspect of documentation, otherwise this is an awesome change. 👍